SConstruct

```diff
@@ -207,6 +207,7 @@ opts.Add(BoolVariable("modules_enabled_by_default", "If no, disable all modules
 opts.Add(BoolVariable("no_editor_splash", "Don't use the custom splash screen for the editor", True))
 opts.Add("system_certs_path", "Use this path as SSL certificates default for editor (for package maintainers)", "")
 opts.Add(BoolVariable("use_precise_math_checks", "Math checks use very precise epsilon (debug option)", False))
+opts.Add("env_vars_to_clone", "A comma separated list of env variables to clone from the outer environment.", "")
```

Comment:
I'd move this a between `vsproj` and `disable_3d`. This seems to be the separation between two sub blocks for advanced options impacting the compiler/environment/output, and options affecting engine logic itself by enabling/disabling stuff.

I'm also not fully sold on the `env_vars_to_clone` name, but I don't really have a better suggestion. `parse_env_vars` maybe, but I'm not fully happy with it either.

Reply:
`import_env_vars`?